examples/run/cuda/seeding_example_cuda.cpp

```diff
@@ -110,11 +114,10 @@ int seq_run(const traccc::opts::track_seeding& seeding_opts,
      * Build a geometry
      *****************************/
 
-    // B field value and its type
-    // @TODO: Set B field as argument
-    const traccc::vector3 B{0, 0, 2 * traccc::unit<traccc::scalar>::T};
-    const traccc::bfield field{
-        traccc::construct_const_bfield<traccc::scalar>(B)};
+    // B field value
+    const traccc::bfield host_field =
+        traccc::details::make_magnetic_field(bfield_opts);
+    const traccc::bfield device_field = traccc::cuda::make_bfield(host_field);
```

Comment:
To point out, this is the relevant bit. To use an inhomogeneous field on a device, one must use the appropriate `make_bfield(...)` function.